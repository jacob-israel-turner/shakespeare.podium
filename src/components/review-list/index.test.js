import React from 'react';
import ReactDOM from 'react-dom';
import ReviewList from './index';
import {StyleSheetTestUtils} from 'aphrodite';

beforeEach(() => {
  StyleSheetTestUtils.suppressStyleInjection();
});

afterEach(() => {
  StyleSheetTestUtils.clearBufferAndResumeStyleInjection();
});

describe('<ReviewList />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ReviewList />, div);
  });
})
