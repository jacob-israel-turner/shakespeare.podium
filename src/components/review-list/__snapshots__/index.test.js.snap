exports[`<ReviewList /> renders without crashing 1`] = `
<div
  className="container_py3uvl">
  <h3>
    Loading...
  </h3>
</div>
`;
